### PR TITLE
completed ticket 5

### PR DESCRIPTION
**Pull request by dpeake99**
Updated the app so that the body is sent. Wrote a test that is passing for updating the votes

### CODE REVIEW

File: controllers/controller.js [hunk 1]
```diff
@@ -29,3 +30,13 @@ exports.customErrorHandler = (err, req, res, next) => {
     console.log(err, "<----- unhandled error");
     res.status(500).send(err);
   };
+
+  exports.updateArticle = (req, res, next) => {
+    const { article_id } = req.params;
+    console.log(req.body)
```
Comment (two-triangles) [inline controllers/controller.js]: don't forget to remove console.logs once you have completed a task :)

File: models/model.js [hunk 2]
```diff
@@ -15,3 +15,12 @@ exports.selectArticleById = (article_id) => {
         } else return result.rows[0];
     })
 }
+
+exports.updateVoteCount = (article_id, inc_votes) => {
+    return db.query("UPDATE articles SET votes = votes + $2 WHERE article_id =$1 RETURNING *", [article_id, inc_votes])
+    .then((result) => {
+        if (result.rows[0] === undefined){
+            return Promise.reject({status: 400, msg: "Invalid article_id"})
```
Comment (two-triangles) [inline models/model.js]: I can see you have implemented the behaviour but I can't see a test for it?

**Comment by two-triangles** [comment]
Looks good, but there should be a test for the error handling you have implemented in your model for this endpoint

**Comment by Dutton1990** [comment]
You could add more error handling on this one: 

invalid id > 400 
inc_vote not an integer > 400

**Pull request merged**